client/src/components/DataFiles/DataFilesSidebar/DataFilesSidebar.scss

```diff
@@ -10,70 +10,29 @@
   border-radius: 0;
 }
 
-#add-button-wrapper {
-  padding-left: var(--horizontal-buffer);
-}
 #data-files-add {
-  width: 174px;
+  width: 100%;
 }
```

Comment:
**What did I do?** Match [design](https://xd.adobe.com/view/654bc6e4-785c-4ed5-81a8-a438fe50ede1-3018/screen/dee4ca3c-4d2b-4bab-ae0c-1cdbb33f9d27/specs/):
- Stretch the button.
- Center the button.